readme.md

```diff
@@ -14,6 +14,15 @@ It's a fancy name for a simple concept: Extending a model (usually to add specif
 composer require "tightenco/parental=0.6"
 ```
 
+Each time you add or remove child classes, you'll want to do the following:
+```bash
+php artisan parental:discover-children
```

Comment:
Yeah, this is just too much work involved for the developer experience. I'll keep reading though.